appvm-scripts/etc/profile.d/qubes-gui.sh

```diff
@@ -1 +1,5 @@
 export DISPLAY=:0 _JAVA_AWT_WM_NONREPARENTING=1
+if ! test -f /var/run/qubes-service/gpu-accel; then
```

Comment:
I'd prefer a name like `software-rendering` or similar, instead of `gpu-accel`. The latter may suggest enabling it will actually get you GPU acceleration, which is not really the case.

Reply:
Sure. I am searching where default services are set to enable `software-rendering` by default... Until I find them, I will commit with the same logic: ` ! test -f /var/run/qubes-service/hardware-rendering`.

Reply:
Found, `vm-systemd/qubes-sysinit.sh`